data/meson.build

```diff
@@ -23,14 +23,14 @@ conf = configuration_data()
 conf.set('dbus_srv_user', dbus_srv_user)
 
 configure_file(
-    output: 'org.containers.hirte.conf',
-    input: 'org.containers.hirte.conf.in',
+    output: 'org.containers.bluechi.conf',
+    input: 'org.containers.bluechi.conf.in',
```

Comment:
org.containers.hirte. should be replaced with io.github.eclipse-bluechi.bluechi according to the doc and today's discussion